### PR TITLE
Allow id to be an int in doc upsert

### PR DESCRIPTION
**Pull request by Westwooo**

### CODE REVIEW

**Comment by chvck** [comment]
The commit message is a bit misleading as i think this applies to all kv store commnands?

File: src/cli/doc_upsert.rs [hunk 1]
```diff
@@ -164,7 +165,18 @@ pub(crate) fn run_kv_store_ops(
             let mut content = None;
             for (k, v) in val.iter() {
                 if k.clone() == id_column {
-                    id = v.as_str().ok();
+                    id = match v {
```
[the remark this answers is not in the document]
Comment (Westwooo) [reply src/cli/doc_upsert.rs:168]: No, get, remove and subdoc_get all use `ids_from_input` which works with ints. 
Replace, upsert and insert all use this so are fine but  import does need changing.

**Comment by ldoguin** [comment]
Found it a bit confusing to have run_kv_store_ops in the Upsert CLI. Would it be better to have a KV_Doc commons with all these operations ?

**Comment by Westwooo** [comment]
> Found it a bit confusing to have run_kv_store_ops in the Upsert CLI. Would it be better to have a KV_Doc commons with all these operations ?

It could be, will raise an issue to look into refactoring this

**Pull request merged**